k8s/jenkins.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
```

Comment:
I'm not qualified to review Jenkins configuration, but could we get some comments in these yaml files to explain what they're for? Running CI for our code? Running WPT? ...?

Reply:
I'll add a README in this directory. These files are from the [GCP docs on how to set up Jenkins on GKE](https://cloud.google.com/solutions/jenkins-on-container-engine).

Reply:
Excellent. A comment + that link would be useful in the yaml file.